### PR TITLE
New player handler: HandlePunchAir

### PR DESCRIPTION
**Pull request by RestartFU**
This allow you to handle when a player punches  the air

### CODE REVIEW

File: server/player/handler.go [hunk 1]
```diff
@@ -169,6 +171,9 @@ func (NopHandler) HandleItemDamage(*event.Context, item.Stack, int) {}
 // HandleAttackEntity ...
 func (NopHandler) HandleAttackEntity(*event.Context, world.Entity, *float64, *float64) {}
 
+//HandlePunchAir ...
```
Comment (Sandertv) [inline server/player/handler.go:174]: Could you add a space after the `//`?

File: server/player/player.go [hunk 2]
```diff
@@ -1938,6 +1938,17 @@ func (p *Player) SwingArm() {
 		v.ViewEntityAction(p, action.SwingArm{})
 	}
 }
+func (p *Player) PunchAir() {
```
Comment (Sandertv) [inline server/player/player.go:1941]: Could you document this?

**Review by Sandertv — COMMENTED** [review]
Looks good save for some documentation things.

**Comment by Sandertv** [comment]
Could you run `go fmt ./...` over the project? The formatting is inconsistent.

File: server/player/player.go [hunk 3]
```diff
@@ -1938,6 +1938,18 @@ func (p *Player) SwingArm() {
...
+// PunchAir makes the player punch the air and plays the attackNoDamage sound.
+func (p *Player) PunchAir() {
+	if p.Dead() {
+		return
+	}
+	ctx := event.C()
+	p.handler().HandlePunchAir(ctx)
+	ctx.Continue(func() {
+		p.SwingArm()
+		p.World().PlaySound(p.Position(), sound.Attack{Damage: false})
```
Comment (Sandertv) [inline server/player/player.go:1950]: You can remove `Damage: false` here as that's the default value.

**Review by RestartFU — COMMENTED** [review]
sorry I wrote rank in my last commit, I meant ran

**Review by Sandertv — APPROVED** [review]
Looks good, thank you for your PR!

**Pull request merged**